src/phase0/state_transition/epoch_processing.rs

```diff
@@ -179,8 +179,28 @@ pub fn process_justification_and_finalization<
         MAX_VALIDATORS_PER_COMMITTEE,
         PENDING_ATTESTATIONS_BOUND,
     >,
-    _context: &Context,
+    context: &Context,
 ) -> Result<(), Error> {
+    // Initial FFG checkpoint values have a `0x00` stub for `root`.
+    // Skip FFG updates in the first two epochs to avoid corner cases that might result in modifying this stub.
+    if get_current_epoch(state, context) > GENESIS_EPOCH + 1 {
+        let previous_attestations =
+            get_matching_target_attestations(state, get_previous_epoch(state, context), context);
+        let current_attestations =
+            get_matching_target_attestations(state, get_current_epoch(state, context), context);
+        let total_active_balance = get_total_active_balance(state, context)?;
+        let previous_target_balance =
+            get_attesting_balance(state, previous_attestations?, context)?;
```

Comment:
note: it is more idiomatic to use the `?` operator at the end of the variable's defn, not at a later use site